docs/content/release_notes/csi_rn_changelog_1.8.0.md

```diff
@@ -0,0 +1,6 @@
+# 1.8.0 (December 2021)
+
+IBM® block storage CSI driver 1.8.0 adds new support and enhancements:
+- Now supports HyperSwap for IBM Spectrum Virtualize Family storage systems
+- New NVMe/FC support
```

Comment:
is `NVMe/FC` is the official name? it might sound like we have some new fc support that is not necessarily related to nvme

Reply:
I'll have a different PR for NVMe stuff. in the meantime i'll see exactly how SVC reers to NVMe and there is official guidance from NVMe site as well. I"ll make an official terminology list for this feature.